contracts/interfaces/ISwapRouter.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: AGPL-3.0
```

Comment:
Can you move this file and all of the other interfaces needed for the allocator into an interfaces folder inside the allocators folder?